templates/firefox-ios.xliff

```diff
@@ -3061,22 +3108,167 @@
+      <trans-unit id="TodayWidget.PocketWidgetSubLabel">
+        <source>Firefox - Recommended by Pocket</source>
+        <note>Sub label for medium size Firefox Pocket stories widge widget. Pocket is the name of another app.</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.PocketWidgetTitle">
+        <source>Recommended by Pocket</source>
+        <note>Title for Firefox Pocket stories widget in Gallery View where user can add it to home screen. Pocket is the name of another app.</note>
+      </trans-unit>
       <trans-unit id="TodayWidget.PrivateTabButtonLabelV1">
         <source>Private Search</source>
         <note>Open New Private Tab button label</note>
       </trans-unit>
+      <trans-unit id="TodayWidget.QuickActionDescription">
+        <source>Select a Firefox shortcut to add to your Home screen.</source>
+        <note>Quick action description when widget enters edit mode</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.QuickActionDropDownMenu">
+        <source>Quick action</source>
+        <note>Quick Actions left label text for dropdown menu when widget enters edit mode</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.QuickActionGalleryDescription">
+        <source>Add a Firefox shortcut to your Home screen. After adding the widget, touch and hold to edit it and select a different shortcut.</source>
+        <note>Description for small size widget to add it to home screen</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.QuickActionsGalleryTitle">
+        <source>Quick Actions</source>
+        <note>Quick Actions title when widget enters edit mode</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.QuickActionsSubLabel">
+        <source>Firefox - Quick Actions</source>
+        <note>Sub label for medium size quick action widget</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.QuickViewGalleryDescription">
+        <source>Access your open tabs directly on your homescreen.</source>
+        <note>Description for Quick View widget in Gallery View where user can add it to home screen</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.QuickViewGalleryTitle">
+        <source>Quick View</source>
+        <note>Title for Quick View widget in Gallery View where user can add it to home screen</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.QuickViewLargeGalleryDescription">
+        <source>Add shortcuts to your open tabs.</source>
+        <note>Description for Quick View widget in Gallery View where user can add it to home screen</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.QuickViewOpenTabsSubLabel">
+        <source>Firefox - Open Tabs</source>
+        <note>Sub label for Top Sites widget</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.SearchLabel">
+        <source>Search</source>
+        <note>This is a common word used in Search in Private Tab and Search in Firefox</note>
```

Comment:
We don't re-use existing translations in other areas of the app, to avoid possible issues in translation.
So, "Search in Private Tab" and "Search in Firefox" should be each of their own strings :)